src/base/digest32.h

```diff
@@ -41,13 +43,15 @@ class Digest32
     using UnderlyingType = lt::digest32<N>;
 
     Digest32() = default;
+    Digest32(const Digest32 &other) = default;
```

Comment:
Don't understand why you need those (and assignment operators) explicitly declared now.
It seems for this class, "rule of zero" still applies.

Reply:
I messed up at first trying to assign it to a constant, and thought the compiler messages meant I should add these operators 😀